Sources/Commands/SwiftTestTool.swift

```diff
@@ -46,6 +47,8 @@ private enum Mode: Argument, Equatable, CustomStringConvertible {
         case "-s", "--specifier":
             guard let specifier = pop() else { throw OptionParserError.expectedAssociatedValue(argument) }
             self = .run(specifier)
+        case "--parallel":
+            self = .runInParallel
```

Comment:
Am I correct in understanding that `--parallel` does not support test filtering? In other words, if I specify `--parallel`, I can't also specify the specific test class or test method I'd like to run?

If that understanding is correct, is there a reason for the limitation? It seems like it should be possible.